tests/src/Functional/CorporateFooterRenderTest.php

```diff
@@ -712,7 +770,9 @@ public function testCorporateFooterRendering(): void {
       'label' => 'Custom contact 1',
       'href' => 'http://example.com/custom-contact-1',
     ];
-    $this->assertListLink($actual, 'standardised', $expected);
+    $this->assertListLink($actual, 'standardised', $expected, TRUE);
+    // We should have the external icon present.
+    $this->assertNotEmpty($actual->find('css', 'svg.ecl-icon.ecl-icon--xs.ecl-link__icon'));
```

Comment:
All the assertions ```// We should have the external icon present.
    $this->assertNotEmpty($actual->find('css', 'svg.ecl-icon.ecl-icon--xs.ecl-link__icon'));``` are not actually checking for the external icon but for any icon. We need to assert that the `svg` tag contains or not the actual `#external` icon for each case.